src/app/main.rs

```diff
@@ -484,6 +484,28 @@ where
                 .layer(strip_header::response::layer(super::L5D_REMOTE_IP))
                 .service(client_stack);
 
+            let balancer = svc::builder()
```

Comment:
Is there a reason this was moved higher up in the file, or is this change unrelated?

Reply:
unrelated, mostly to make it easier to comment pieces out